app/ide-desktop/lib/performance-logger/src/performance_logger.ts

```diff
@@ -0,0 +1,245 @@
+export enum LogLevel {
+    Section = 'SECTION',
+    Task = 'TASK',
+    Detail = 'DETAIL',
+    Debug = 'DEBUG',
+}
+
+/// Delimiter used to to encode information in the `PerformanceEntry` name.
+const MESSAGE_DELIMITER = '//'
+
+const ENABLE_SECTION_PROFILING = process.env.PROFILING_LEVEL.toLowerCase().includes('section')
+const ENABLE_TASK_PROFILING = process.env.PROFILING_LEVEL.toLowerCase().includes('task')
+const ENABLE_DETAIL_PROFILING = process.env.PROFILING_LEVEL.toLowerCase().includes('detail')
+const ENABLE_DEBUG_PROFILING = process.env.PROFILING_LEVEL.toLowerCase().includes('debug')
```

Comment:
These lines do not have any explanation. Why do we check `process.env`, what does it do?

Reply:
Added comments.